src/executors/priority_executor.rs

```diff
@@ -34,6 +34,7 @@ static GWEI_PER_ETH: U256 = u256!(1_000_000_000);
 const QUOTE_ETH_LOG10_THRESHOLD: usize = 8;
 // The number of bps to add to the base bid for each fallback bid
 const DEFAULT_FALLBACK_BID_SCALE_FACTOR: u64 = 50;
+const CONFIRMATION_TIMEOUT: u64 = 30;
```

Comment:
this seems like an extremely long timeout -- what are the reason(s)?

Reply:
Oops, this was supposed to be 10s.